ADOPTERS.md

```diff
@@ -15,6 +15,9 @@ This is a list of adopters of using PowerShell in production or in their product
 * [Azure Cloud Shell](https://shell.azure.com/) provides a batteries-included browser-based PowerShell environment used by Azure administrators to manage their environment.
   It includes up-to-date PowerShell modules for `Azure`, `AzureAD`, `Exchange`, `Teams`, and many more.
   More information about Azure Cloud Shell is available at [Azure Cloud Shell Overview.](https://docs.microsoft.com/azure/cloud-shell/overview)
+* [Azure Functions - PowerShell](https://github.com/Azure/azure-functions-powershell-worker) is a serveless compute service to execute PowerShell scripts in the cloud without worrying about managing resources.
```

Comment:
```suggestion
* [Azure Functions - PowerShell](https://github.com/Azure/azure-functions-powershell-worker) is a serverless compute service to execute PowerShell scripts in the cloud without worrying about managing resources.
```

Reply:
Thanks @SteveL-MSFT!

Reply:
Is the build failing because the repo name contains `powershell` in lower case?

![image](https://user-images.githubusercontent.com/12720858/70580931-4cf79a00-1b6a-11ea-9022-9f841bb39e14.png)